content/docs/quickstart/azure/destroy-stack.md

```diff
@@ -15,7 +15,7 @@ To destroy resources, run the following:
 $ pulumi destroy
 ```
 
-You'll be prompted to make sure you really want to delete these resources. This takes some time; Pulumi waits for the ACI to shutdown and for the resource group to be removed before it considers the destroy operation to be complete.
```

Comment:
It's annoyingly pedantic, but IMO a semicolon here kinda works, since that the sentence that follows it explains why it takes some time -- although what `this` refers to remains a bit unclear. (Does the prompting take time, or the deletion?) So maybe `Deletion takes some time, as Pulumi waits...` or `, which takes some time, as Pulumi waits...` But again, this is totally minor, feel free to ignore, etc. -- I'm just a bit of a grammar nerd. 🤓

Reply:
I love this suggestion though!